drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -1993,6 +1993,27 @@ void RasterizerSceneGLES3::render_scene(const Ref<RenderSceneBuffers> &p_render_
 		_draw_sky(render_data.environment, render_data.cam_projection, render_data.cam_transform, sky_energy_multiplier, p_camera_data->view_count > 1, flip_y);
 	}
 
+	if (scene_state.used_screen_texture || scene_state.used_depth_texture) {
+		texture_storage->copy_scene_to_backbuffer(rt, scene_state.used_screen_texture, scene_state.used_depth_texture);
+		glReadBuffer(GL_COLOR_ATTACHMENT0);
```

Comment:
```suggestion
		texture_storage->copy_scene_to_backbuffer(rt, scene_state.used_screen_texture, scene_state.used_depth_texture);
		glBindFramebuffer(GL_READ_FRAMEBUFFER, rt->fbo);
		glReadBuffer(GL_COLOR_ATTACHMENT0);
```
We need to specify the READ_FRAMEBUFFER or else it can try to read from the last buffer specified as the READ_FRAMEBUFFER